### PR TITLE
IGNITE-14714 LineLength checkstyle rule added

### PR DESCRIPTION
**Pull request by nizhikov**
This PR adds `LineLength` rule to check style and fixes violations in the current codebase.

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

**Review by Sega76 — APPROVED** [review]

**Comment by ololo3000** [comment]
I propose to add <option name="RIGHT_MARGIN" value="140"/> to ignite/idea/ignite_codeStyle.xml. The default is 120. 
WDYT?

**Comment by nizhikov** [comment]
@ololo3000 Good cactch. Thanks. I've added this to the configuration.

File: modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/internal/ConnectionClientPool.java [hunk 1]
```diff
@@ -670,8 +679,11 @@ public boolean removeNodeClient(UUID nodeId, GridCommunicationClient rmvClient)
         for (; ; ) {
             GridCommunicationClient[] curClients = clients.get(nodeId);
 
-            if (curClients == null || rmvClient.connectionIndex() >= curClients.length || curClients[rmvClient.connectionIndex()] != rmvClient)
+            if (curClients == null
+                || rmvClient.connectionIndex() >= curClients.length
+                || curClients[rmvClient.connectionIndex()] != rmvClient) {
```
Comment (ololo3000) [inline modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/internal/ConnectionClientPool.java:684]: The next statement is one-liner. We don't need braces here.

Comment (nizhikov) [reply modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/internal/ConnectionClientPool.java:684]: Fixed

**Review by ololo3000 — APPROVED** [review]

**Pull request merged**